src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -26,7 +26,7 @@
 				:button-variant="true"
 				:checked="activeTab === tab.id"
 				:wrapper-id="`tab-button-${tab.id}`"
-				:tabindex="activeTab === tab.id ? 0 : -1"
+				:tabindex="activeTab === tab.id && !showForSingleTab ? 0 : -1"
```

Comment:
Is it needed? Can we have a single tab with a different active tab?

Reply:
The point was to skip a navigation (as part of event disabling). But if we'll decide to keep it, then I'll simply revert this change

Reply:
Technically it is still an active interactive element